gtk/Graphics/UI/Gtk/Cairo.chs

```diff
@@ -52,8 +52,8 @@ module Graphics.UI.Gtk.Cairo (
   -- * Functions for the 'Render' monad.
 #if GTK_MAJOR_VERSION < 3
   renderWithDrawable,
-  region,
 #else
+  region,
```

Comment:
Ooops, now it is not exported for `GTK_MAJOR_VERSION < 3`

Reply:
Sorry, it's now fixed.